### DISCUSSION TITLE
about RenderSchedule.ScriptProps

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3848] [Q&A] [answered]
now , i want upate elementid transfrom display,but there have 2 problem:
1.the all model element  do transfrom?
2.there have new transfrom model display,but i thank   the init model notl display in the time at transformTimeline value?

![transfrom](https://user-images.githubusercontent.com/18022720/175051176-a0f05159-e827-431d-b896-907914522b32.png)

 const _data: RenderSchedule.ScriptProps = [
            {
                modelId: "0x15",
                elementTimelines: [{
                    batchId: 1,
                    elementIds: ["0x522"],
                    transformTimeline:
                        [
                            {
                                interpolation: 2,
                                time: 200,
                                value: {
                                    transform: [
                                        [
                                            1,
                                            0,
                                            0,
                                            -100
                                        ],.......

let style: DisplayStyleSettingsProps = { 
                scheduleScript: data(), 
            }  
            _vp.timePoint=300; 
            _vp.overrideDisplayStyle(style);

**Comment by pmconne** [comment, marked as answer]
We are [working on](https://github.com/iTwin/itwinjs-core/pull/3428) making it possible to edit the schedule script in frontend code, but currently the script must be defined on an element stored in the iModel. So what you are attempting to do will not work until #3428 is complete.

I do not fully understand your numbered questions.

**Reply by paprophet** [reply]
ok, wait you finish the work about edit the schedule script in frontend code.
my problem is  
_vp.timePoint = 300 
1.in script elementIds give the value,so should show the changed element in elementIds ,but now model elements all tranform  
2. vp display the new tranform element ,and the pre-time model should not show
